src/wp-cron.php

```diff
@@ -19,7 +19,7 @@
 ignore_user_abort( true );
 
 /* Don't make the request block till we finish, if possible. */
-if ( function_exists( 'fastcgi_finish_request' ) && version_compare( phpversion(), '7.0.16', '>=' ) ) {
+if ( PHP_VERSION_ID >= 70016 && function_exists( 'fastcgi_finish_request' ) ) {
```

Comment:
Is there a reason for reversing the order of the conditions here?

I think `function_exists( 'fastcgi_finish_request' )` is the "primary" condition here and should probably remain first, as the second one can be removed once the minimum supported PHP version is bumped to 7.1+.

Reply:
Thanks! To me, that makes the condition slightly less readable, as the main check is now placed behind the supplemental one.

So I guess I'm not a fan of this specific micro-optimization, but I see the point and don't have any strong objections if that's preferred here 🙂